core/scripts/chaincli/handler/handler.go

```diff
@@ -0,0 +1,67 @@
+package handler
+
+import (
+	"crypto/ecdsa"
+	"log"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethclient"
+
+	link "github.com/smartcontractkit/chainlink/core/internal/gethwrappers/generated/link_token_interface"
+	"github.com/smartcontractkit/chainlink/core/scripts/chaincli/config"
+)
+
+// baseHandler is the common handler with a common logic
```

Comment:
do we need to track a `TODO` to revisit for multi chain?

Reply:
It should be compatible with all EVM chains